### ISSUE TITLE
Using forked repos for comparison branch in `sl pr submit`

### ISSUE THREAD

**Issue by jaymzh** [facebook/sapling#538]
`sl pr submit` seems to push a branch to the upstream repo and not the fork. Which may not work if you don't have access to the upstream repo. I can't even find a way to tell it about my fork. I've reviewed [PR docs](https://sapling-scm.com/docs/commands/pr), [Sapling Stack](https://sapling-scm.com/docs/git/sapling-stack/) docs, and [config docs](https://sapling-scm.com/docs/commands/config).

Eventually I found `sl paths` in [Git cheatsheet](https://sapling-scm.com/docs/introduction/git-cheat-sheet), and then `sl paths --help`, eventually lead me to:

```
sl paths -a default-push <path>
```

Which seems like it should work, but it does not:

```
[phil@rider scale_chef_client]$ sl paths -a default-push git@github.com:jaymzh/scale-chef.git
normalized path git@github.com:jaymzh/scale-chef.git to ssh://git@github.com/jaymzh/scale-chef.git
[phil@rider scale_chef_client]$ sl pr submit
pushing 1 to ssh://git@github.com/socallinuxexpo/scale-chef.git
```

Sad Panda.

1. A away to do this
2. Docs on how to do this
3. `sl clone` should probably have an option to set this up automatically. It already knows my GH username, so it can already see if I have a fork, and set it up appropriately. That said, to follow the philosophy of sl, we probably want something like `sl clone -f <url>` or some-such.

**Comment by jaymzh** [comment]
Updated the above - the way that seemed like it would work, did not.

**Comment by jaymzh** [comment]
@muirdm - is there a work around?

**Comment by muirdm** [comment]
This should be resolved as of b31704fa68c712bbcb587d5ea583de5bfadb8c62. We are working on getting another release out now.

**Comment by jaymzh** [comment]
@muirdm - Many thanks! As a former (9-year) FB/Meta employee, I know that sometimes it can be hard to get time to work on Issues from the PSS community, and I'm very grateful for how responsive the Sapling team has been since the release!
